test/e2e/apps/rc.go

```diff
@@ -41,6 +46,14 @@ import (
 var _ = SIGDescribe("ReplicationController", func() {
 	f := framework.NewDefaultFramework("replication-controller")
 
+	var ns string
+	var dc dynamic.Interface
+
+	ginkgo.BeforeEach(func() {
+		ns = f.Namespace.Name
```

Comment:
nit: All existing tests in this rc.go file are using `f.Namespace.Name` in these functions.
So it is nice to use `f.Namespace.Name` directly like the other tests and create another cleanup PR if really necessary to cover the other tests.
This comes into my mind because here is a common part for all tests in this rc.go file but this `ns` is only used in additional test code.